charts/quickwit/templates/servicemonitor.yaml

```diff
@@ -5,6 +5,9 @@ metadata:
   name: {{ include "quickwit.fullname" . }}
   labels:
     {{- include "quickwit.labels" . | nindent 4 }}
+    {{- with .Values.serviceMonitor.additionalLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

Comment:
do we really need an extra set of additionalLabels? Can't the Prometheus operator be configured to use the `quickwit.labels` which are already configurable?

Reply:
It's a common pattern in the Prometheus Operator community to have these labels for ServiceMonitor, PodMonitor, PrometheusRule, ...
Ex:
- https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-elasticsearch-exporter/templates/prometheusrule.yaml
- https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-elasticsearch-exporter/templates/podmonitor.yaml
- https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-elasticsearch-exporter/templates/servicemonitor.yaml